app/controllers/v1/open_api_specification_controller.rb

```diff
@@ -0,0 +1,184 @@
+module V1
+  class OpenApiSpecificationController < ApplicationController
+    include Swagger::Blocks
+
+    swagger_root do
+      key :swagger, '2.0'
+
+      info version: '1.0.0' do
+        key :title, 'URI Service API'
+        key :description, 'API to manage vocabularies and their associated terms.'
+      end
+
+      key :basePath, '/api/v1'
+      key :consumes, ['application/json']
+      key :produces, ['application/json']
+
+      security_definition :api_key, type: :apiKey do
+        key :name, :api_key
+        key :in, :header
+      end
+    end
+
+    swagger_path '/vocabularies' do
+      operation :get do
+        key :description, 'Returns paginated vocabularies'
+
+        parameter name: :per_page, in: :query, type: :number,
+                  minimum: 1, maximum: 500,
```

Comment:
Can we have 500 come from our MAX_PER_PAGE constant instead of being a hard-coded value?

Reply:
Yes!